exercises/ocr-numbers/ocr_numbers_test.py

```diff
@@ -1,146 +1,122 @@
-"""Tests for the ocr-numbers exercise
-
-Implementation note:
-ocr.convert should validate its input and
-raise ValueErrors with meaningful error messages
-if necessary.
```

Comment:
This content should go in a new `.meta/hints.md`

Reply:
The exercise readme will need to be [regenerated](https://github.com/exercism/python/blob/master/CONTRIBUTING.md#generating-exercise-readmes) after hints.md is created/modified.

Reply:
I did that, can you check?